File: ReformaAgraria/Models/ToraMapAttribute.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ReformaAgraria.Models
{
    public class ToraMapAttribute: BaseEntity<int>
    {
        public ToraMapAttribute() { }

        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public override int Id { get; set; }

        public decimal Size { get; set; }

        public ToraSettingProcessStage ToraSettingProcessStage { get; set; }

        public Status BorderSettingStatus { get; set; }

        public string Attachment { get; set; }

        public int? FkCoordinateId { get; set; }

        [ForeignKey("FkCoordinateId")]
        public Coordinate Coordinate { get; set; }

        public string FkRegionId { get; set; }

        [ForeignKey("FkRegionId")]
        public Region Region { get; set; }

    }
}

File: ReformaAgraria/ClientApp/app/models/ToraMapAttribute.ts
import { ToraSettingProcessStage } from './ToraSettingProcessStage';
import { Status } from './Status';
import { Coordinate } from './Coordinate';
import { Region } from './Region';
import { BaseEntity } from './BaseEntity';

export interface ToraMapAttribute extends BaseEntity<number> { 
	id: number;
	size: number;
	toraSettingProcessStage: ToraSettingProcessStage;
	borderSettingStatus: Status;
	attachment: string;
	fkCoordinateId: number;
	coordinate: Coordinate;
	fkRegionId: string;
	region: Region;
}
